internal/schemautil/userconfig/apiconvert/toapi.go

```diff
@@ -17,6 +17,16 @@ type resourceDatable interface {
 	IsNewResource() bool
 }
 
+var (
+	// dotAnyDotNumberRegExp is a regular expression that matches a string that begins with a dot and ends with a dot
+	// and a number.
+	dotAnyDotNumberRegExp = regexp.MustCompile(`.+\.[0-9]$`)
+
+	// dotNumberEOLOrDotRegExp is a regular expression that matches a string that begins with a dot and ends with a
+	// number and an end of line or a dot.
+	dotNumberEOLOrDotRegExp = regexp.MustCompile(`\.\d($|\.)`)
```

Comment:
It matches "dot, number, dot" anywhere and "dot, number" at the end of the line. Is that expected?